README.md

```diff
@@ -32,18 +32,18 @@ Implementation details
   * Structured to facilitate review and analysis.
   * Intended to be portable to any system with a C89 compiler and uint64_t support.
   * No use of floating types.
-  * Expose only higher level interfaces to minimize the API surface and improve application security. ("Be difficult to use insecurely.")
-* Field operations
+  * Expose only higher level interfaces to minimize the API surface and improve application security. ("Be difficult to use insecurely").
+* Field operations.
   * Optimized implementation of arithmetic modulo the curve's field size (2^256 - 0x1000003D1).
-    * Using 5 52-bit limbs
+    * Using 5 52-bit limbs.
     * Using 10 26-bit limbs (including hand-optimized assembly for 32-bit ARM, by Wladimir J. van der Laan).
       * This is an experimental feature that has not received enough scrutiny to satisfy the standard of quality of this library but is made available for testing and review by the community.
-* Scalar operations
+* Scalar operations.
```

Comment:
Agree with @real-or-random. Adding periods in places like this does not make sense.

Reply:
While you're 100% correct that there might not be technical merit in these miniscule changes, it just irked me a little bit, that the words that described the technicals, weren't 100% punctuated, and as a grammar expert, I felt compelled to do the needful, for other grammarians in the future who'd probably come across this document.

Reply:
Sorry, I don't get your comment. Closing this PR for now. Feel free to reopen if you have an updated suggestion.